alf/algorithms/ppo_loss.py

```diff
@@ -89,9 +85,9 @@ def __init__(self,
             action_spec=action_spec,
             gamma=gamma,
             td_error_loss_fn=td_error_loss_fn,
-            use_gae=use_gae,
+            use_gae=True,
             td_lambda=td_lambda,
-            use_td_lambda_return=use_td_lambda_return,
+            use_td_lambda_return=True,
```

Comment:
Potentially we can also disable gae or td_lambda for off-policy advantage estimation. Maybe leave this to future.